src/state/slices/portfolioSlice/utils.ts

```diff
@@ -111,8 +111,10 @@ export const accountIdToLabel = (accountId: AccountSpecifier): string => {
 }
 
 // note - this is not really a selector, more of a util
-export const accountIdToFeeAssetId = (accountId: AccountSpecifier): AssetId =>
-  chainIdToFeeAssetId(accountIdToChainId(accountId))
+export const accountIdToFeeAssetId = (accountId: AccountSpecifier): AssetId => {
+  const chainId = accountIdToChainId(accountId)
+  return getChainAdapters().get(chainId)?.getFeeAssetId() as AssetId
```

Comment:
few thoughts in general about this PR - there exists a pure function mapping a chainId to a feeAsset.

* this could belong in caip as a util
* we could make the chain adapters exhaustive - as i don't like `?.getFeeAssetId() as AssetId` - if it's undefined we shouldn't cast it

Reply:
Discussed this one out on Discord, we now have all we need to get this from adapters rather than a mapping in caip, and accessing the adapters list is now cheap with the singleton.

Reply:
Casting removed in 99127ce1cc091fe62da1f881eb79635b11c2ddbc - properly handling undefined everywhere